### PR TITLE
fix: align dependency versions

### PR DESCRIPTION
**Pull request by rflechtner**
Fixes minor incoherences in dependencies that are shared between multiple sub-packages.

### CODE REVIEW

File: docs/package.json [hunk 1]
```diff
@@ -17,11 +17,11 @@
   "homepage": "https://github.com/KILTprotocol/sdk-js#readme",
   "dependencies": {
     "@kiltprotocol/sdk-js": "workspace:*",
-    "@polkadot/keyring": "^10.0.0",
+    "@polkadot/keyring": "^10.2.1",
```
Comment (rflechtner) [inline docs/package.json:20]: all 9.10.2 versions of the api packages require at least 10.2.1 of the utils packages

File: packages/augment-api/package.json [hunk 2]
```diff
@@ -38,12 +38,12 @@
...
     "@polkadot/api": "^9.10.2",
     "@polkadot/typegen": "^9.10.2",
-    "@types/websocket": "^1.0.0",
+    "@types/websocket": "^1.0.5",
     "glob": "^7.1.1",
     "rimraf": "^3.0.2",
     "ts-node": "^10.4.0",
     "typescript": "^4.8.3",
-    "websocket": "^1.0.31",
+    "websocket": "^1.0.34",
```
Comment (rflechtner) [inline packages/augment-api/package.json:46]: aligning with polkadot/typegen

File: packages/chain-helpers/package.json [hunk 3]
```diff
@@ -31,7 +31,7 @@
   "homepage": "https://github.com/KILTprotocol/sdk-js#readme",
   "devDependencies": {
     "@kiltprotocol/testing": "workspace:*",
-    "@polkadot/keyring": "^10.0.0",
+    "@polkadot/keyring": "^10.2.1",
```
Comment (rflechtner) [inline packages/chain-helpers/package.json:34]: see above

File: packages/did/package.json [hunk 4]
```diff
@@ -40,11 +40,11 @@
...
-    "@polkadot/keyring": "^10.0.0",
+    "@polkadot/keyring": "^10.2.1",
     "@polkadot/types": "^9.10.2",
     "@polkadot/types-codec": "^9.10.2",
-    "@polkadot/util": "^10.0.0",
-    "@polkadot/util-crypto": "^10.0.0",
-    "cbor": "^8.0.2"
+    "@polkadot/util": "^10.2.1",
+    "@polkadot/util-crypto": "^10.2.1",
+    "cbor": "^8.1.0"
```
Comment (rflechtner) [inline packages/did/package.json:48]: the core package requires at least 8.1.0

**Review by ntn-x2 — APPROVED** [review]
LGTM! I was wondering, is there any tool that works like [Cargo root workspace](https://doc.rust-lang.org/cargo/reference/workspaces.html#the-dependencies-table) does, which defines all dependencies and their versions only once?

**Review by arty-name — APPROVED** [review]

**Comment by arty-name** [comment]
I think this is one of those problems that would be inconceivable with the good old single package approach :)

**Comment by rflechtner** [comment]
> LGTM! I was wondering, is there any tool that works like [Cargo root workspace](https://doc.rust-lang.org/cargo/reference/workspaces.html#the-dependencies-table) does, which defines all dependencies and their versions only once?

Well that's at odds with the idea of having multiple packages with multiple package.json's defining dependency ranges, isn't it? We have one lock file in the root workspace, but I think that's as close to this as it gets.

**Comment by rflechtner** [comment]
> I think this is one of those problems that would be inconceivable with the good old single package approach :)

I knooooowwww, one of these days... it's quite a big breaking change though, so probably not happening anytime soon.

**Pull request merged**